src/components/PrintForm/ScaleSelect/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {
+  useEffect,
+  useState
+} from 'react';
+
+import {
+  Select
+} from 'antd';
+import {
+  SelectProps
+} from 'antd/lib/select';
+
+import {
+  MapFishPrintV3Manager
+} from '@terrestris/mapfish-print-manager';
+
+import useMap from '@terrestris/react-geo/dist/Hook/useMap';
+
+export interface ScaleSelectProps extends SelectProps<number> {
+  printManager: MapFishPrintV3Manager;
+}
+
+export const ScaleSelect: React.FC<ScaleSelectProps> = ({
+  printManager,
+  placeholder,
+  value,
+  ...restProps
+}): JSX.Element => {
+
+  const map = useMap();
+
+  const [scale, setScale] = useState<any>(value);
```

Comment:
Any chance to get rid of the `any` type?

Reply:
done with 88360d8